judge/bridge/judgecallback.py

```diff
@@ -351,14 +351,15 @@ def on_test_case(self, packet, max_feedback=SubmissionTestCase._meta.get_field('
         test_case.total = packet['total-points']
         test_case.batch = self.batch_id if self.in_batch else None
         test_case.feedback = (packet.get('feedback', None) or '')[:max_feedback]
+        test_case.extended_feedback = packet.get('extended_feedback', None) or ''
```

Comment:
`dict.get(x)` returns `None` when not found, so the second argument is hardly necessary.